host_types.go

```diff
@@ -132,6 +132,8 @@ type HostCreate struct {
 	Name *string `json:"name"`
 	// os type
 	OsType *OSTypeEnum `json:"os_type"`
+
+	Metadata *map[string]string `json:"metadata,omitempty"`
```

Comment:
Please add the comment with this format, Otherwise lint checks will fail `// Metadata`

Reply:
Done